src/OpenIDDiscoveryCertificateProvider.php

```diff
@@ -199,16 +199,7 @@ private function parseJWKSToCertsHash(ResponseInterface $keys): array
     {
         $keys_json = \GuzzleHttp\json_decode($keys->getBody(), TRUE);
 
-        $key_resources = JWK::parseKeySet($keys_json);
-        $certs         = [];
-        foreach ($key_resources as $kid => $cert) {
-            $certs[$kid] = openssl_pkey_get_details($cert)['key'];
-            if (PHP_MAJOR_VERSION < 8) {
-                openssl_pkey_free($cert);
-            }
-        }
-
-        return $certs;
+        return JWK::parseKeySet($keys_json);
```

Comment:
Just needs an update to the docs (phpdoc & words) for the top-level public `getCertificates` to reflect that we're now returning an array of Keys, not `The returned data is a simple hash of 'key_id (kid)' => '{certificate string in PEM format}'`